### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by nitinmotgi**
Updating plugin documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,13 +373,12 @@ applications or building plugins for appplications. You can expect questions fro
...
+## IRC Channel
+
 CDAP IRC Channel: #cdap on irc.freenode.net
 
 
-License and Trademarks
-======================
+## License and Trademarks
 
 Copyright © 2015-2016 Cask Data, Inc.
```
Comment (abhinavbansal246) [inline README.md]: 2015-2017

Comment (nitinmotgi) [reply README.md]: done!

**Comment by nitinmotgi** [comment]
@abhinavbansal246 can I merge the PR ?

**Comment by nitinmotgi** [comment]
Merging it!

**Pull request merged**